apps/web/app/reschedule/[uid]/page.tsx

```diff
@@ -1,17 +1,10 @@
 import { withAppDirSsr } from "app/WithAppDirSsr";
 import type { PageProps } from "app/_types";
-import { _generateMetadata } from "app/_utils";
 import { headers, cookies } from "next/headers";
 
 import { buildLegacyCtx } from "@lib/buildLegacyCtx";
 import { getServerSideProps } from "@lib/reschedule/[uid]/getServerSideProps";
 
-export const generateMetadata = async () =>
```

Comment:
The metadata generation would never be used since the user will be redirected before the page renders.